source/release-notes.txt

```diff
@@ -9,6 +9,12 @@ Release Notes
    :backlinks: none
    :depth: 1
    :class: twocols
+
+|compass| 1.22
+--------------
+
+- Added an :ref:`embedded MongoDB Shell <embedded-mongodb-shell>`. You
+  can now test queries and operations with your database. 
```

Comment:
[suggestion] This second sentence makes it sound like you couldn't test queries / operations in your database in Compass before, which isn't really true. You could always test queries / operations, now you just have a new way of doing them.

I would consider rewording this sentence slightly to emphasize that you can _use the new shell_ to test queries and operations.

Consider something like:

```
Use the MongoDB Shell to test queries and run operations in an interactive JavaScript interface.
```